### DISCUSSION TITLE
Logout redirect

### DISCUSSION THREAD

**Discussion by iDigAcademy** [laravel/nova-issues#4831] [Q&A] [answered]
Is there a way to change the logout redirect?
Nova 4 is installed in a Laravel 9 application with it's own Auth. Upon logging out via Nova, a user is sent to the Nova login form. I'd rather send it to the application login form.

**Comment by juse-less** [comment, marked as answer]
See [this answer](https://github.com/laravel/nova-issues/discussions/3954#discussioncomment-2569596)  for more details on custom authentication paths.

I tested them a bit and [wrote down](https://github.com/laravel/nova-issues/discussions/3954#discussioncomment-3592620)  my notes on what I found.
